### PR TITLE
Proposal v0.1

### PR DESCRIPTION
**Pull request by victorgcramos**

### CODE REVIEW

File: proposal-v1.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+The previous BR marketing proposal hasn't delivered what the stakeholders were
+expecting, and we all know Decred community members are really expecting
+results, and this is why I am putting myself as a supervisor for this proposal.
+I don't have a clear idea about what caused the previous proposal to fail. In
+fact, noticing a lack of community contributions, I contacted Decred BR old
+admins to try a new social media communication approach in November 2021, and
+took control of Instagram and Twitter accounts, as well as admin rights on
+Facebook group.
+
+The fact that I, @victorguedes, am involved on "Politeia Q1 2022" proposal
```
Comment (victorgcramos) [inline proposal-v1.md:60]: mention `@victorguedes` at the beginning and remove here
```suggestion
The fact that I am involved on "Politeia Q1 2022" proposal
```

File: proposal-v1.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+blocked me from having time to frequently produce contents, so I contacted
+@jpsantanna to handle that responsibility and focus on publishing content almost
+every day on Instagram and Twitter.
```
Comment (victorgcramos) [inline proposal-v1.md:63]:
```suggestion
@jpsantanna to handle that responsibility and focus on reactivating Instagram and Twitter accounts.
```

File: proposal-v1.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+Our first goal was to reactivate the Brazilian community by actively posting
+news, project updates and engagement content in a way that the community could
+notice our return.
+
+After stablishing a good frequency for content production, we decided to expand
+the presence of the BR community on some relevant crypto events. @edsondecre
+took the responsibility to actively participate on those events on Apr 2022,
+and since then, we were able to participate in some meetups and in some of the
```
Comment (victorgcramos) [inline proposal-v1.md:72]: which meetups?

File: proposal-v1.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+most popular Brazilian Crypto events:
+
+- Bitconf IX (we also promoted a ticket giveaway for BR community members!!
+  (https://www.instagram.com/p/CdWkO2JueDU/));
```
Comment (victorgcramos) [inline proposal-v1.md:76]: we promoted more than one! Link all

Link photos, tweets and everything!!!

File: proposal-v1.md [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+- Cryptominds III (​​https://www.instagram.com/p/CdG9jL5vnu4/);
+
+@eric_vieira also joined us on this meantime. He is the admin of the Decred
+Brazil Facebook group, and took the responsibility to post and interact with
+the community there.
```
Comment (victorgcramos) [inline proposal-v1.md:81]: Eric has also translated many posts and articles!

File: proposal-v1.md [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+  and Twitter accounts since November 2021.
+- @edsondecre has worked on several events for Decred, such as Campus Party,
+  Bitconf, and other meetups, with a high-quality community management
+  experience. Edson was able to include Decred in several exchanges like:
+  Bitcambio (https://www.bitcambio.com.br/) and ComprarBitcoin
+  (https://comprarbitcoin.com.br/).
+- @eric_vieira, moderator of DECRED Brasil (Facebook) since February 6, 2017,
+  has been actively explaining technical parts of the project on Facebook
+  Group and also organized a speech about Decred project in Centro Universitário
+  Estácio Juiz de Fora.
```
Comment (victorgcramos) [inline proposal-v1.md:99]: when? sources!!

File: proposal-v1.md [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+#### Level 3
+
+Rate: $30/hr
+
+The last and top level of Decred Brazil contribution journey. A contributor from
+this level is experienced enough to coordinate the team along with other Level 3
+contributors.
+
+##### Expected Roles
+
```
Comment (victorgcramos) [inline proposal-v1.md:157]: Level 2 roles, also.

File: proposal-v1.md [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+- Level 1 expected roles;
+- Noticeable presence on events, coordinating at least one aspect (speech,
+  interaction dynamics...);
+- Complex content production like long videos, podcasts, interviews, articles,
+  tutorials;
+- Support and help the community to handle some possible issues, as well as
+  report the feedbacks using the correct mean of communication with Decred devs
```
Comment (victorgcramos) [inline proposal-v1.md:142]:
```suggestion
  reporting feedbacks using the correct means of communication with Decred devs
```

File: proposal-v1.md [hunk 9]
```diff
@@ -0,0 +1,221 @@
...
+  marketing campaigns, and provide a report document periodically.
+- Supervise Level 1 contributors
+
+#### Level 3
+
+Rate: $30/hr
+
+The last and top level of Decred Brazil contribution journey. A contributor from
+this level is experienced enough to coordinate the team along with other Level 3
+contributors.
```
Comment (victorgcramos) [inline proposal-v1.md:154]: Level 3 must be an influent character on the crypto scenario, experienced with marketing campaigns and metrics....

File: proposal-v1.md [hunk 10]
```diff
@@ -0,0 +1,221 @@
...
+A supervisor is not an active contributor of the social media, but it helps
+other contributors on content creation and other roles described above.
+
+## Budget
+
+A rate of **$17/hr** will be used as an average for this budget estimate, with
+an estimated workload of **15h/week** for each contributor, which results in a
+rounded amount of **$18,500.00** budget for labor.
+
+We are also adding an amount of $200 for marketing campaigns and ad boosts.
```
Comment (victorgcramos) [inline proposal-v1.md:187]: monthly amount!

File: proposal-v1.md [hunk 11]
```diff
@@ -0,0 +1,221 @@
...
+
+@edsondecre has provided a good report about the results of a monthly campaign
+on ComprarBitcoin, which is detailed here: (Link) <!-- TODO: Update the link -->
+
+| Type            | Amount ($USD) |
+| --------------- | ------------- |
+| Labor           | 18,500.00     |
+| Ads             | 1,000.00      |
```
Comment (victorgcramos) [inline proposal-v1.md:195]: 1200. Also add supervisor amount

**Review by victorgcramos — COMMENTED** [review]
reminders

File: proposal-v1.md [hunk 12]
```diff
@@ -0,0 +1,221 @@
...
+Rate: $80/week
+
+Responsible for reviewing invoices, controlling the budget, and providing
+updates about the work done for each month using the Proposal Update section
+on Politeia.
+
+The main role of the supervisor is to provide transparency for the budget spent
+on the proposal.
+
+A symbolic rate of $80/week is enough to cover the time spent on the described
```
Comment (tiagoalvesdulce) [inline proposal-v1.md:175]: Just mention the value. No need to say it's symbolic.

**Review by tiagoalvesdulce — COMMENTED** [review]
Looking at the milestones I assume this proposal is for 6 months, right?

Left 1 comment as feedback and the budget is a bit confusing. Also, I'm not familiar with how social media works in general but don't you think that going for 3 tweets/instagram posts/day is too much? I mean, the content can become boring and repetitive.

As a final suggestion, would be awesome if one or more contributors of the proposal could translate the Decred Journal and the Politeia Digest. Exitus just opened a proposal to continue them: https://proposals.decred.org/record/4fdef29. Since I think it's a good source to take ideas for Decred related posts, translating the full articles won't be that much work. Keep in mind that Kozel has [a proposal for content translation](https://proposals.decred.org/record/7057e0b) where Marcelo Martins from [stakey.club](https://stakey.club/pt/articles/) is the PT translator but I think stakey.club is abandoned and very few editions of the Decred Journey were translated to PT as can be seen here: https://xaur.github.io/decred-news/. Not sure how this would play out. I left a comment on Kozel's proposal here: https://proposals.decred.org/record/7057e0b/comments/1.

File: proposal-v1.md (hunk 4 above)
Comment (victorgcramos) [reply proposal-v1.md:76]: 2500 attendees!!

**Pull request merged**